test/scripts/local.txtar

```diff
@@ -4,14 +4,9 @@ exec cp $WASM_FILE .
 # [execute] call the bky-as CLI to attest a function call
 stdin fn-call.json
 exec bky-as attest-fn-call
-cp stdout result.json
 
-# [check] assert stderr and stdout expected values
+# [check] assert expected values
```

Comment:
I couldn't add a comment any closer... Why is the nitro debug measurement in the list of acceptable measurements?

Reply:
It was the sample config from Delphi. I updated to just the minimal config required to run.